src/components/Inputs/CheckboxInput/CheckboxInput.js

```diff
@@ -39,12 +39,19 @@ export const CheckboxInput = ({
 
   const onChange = (ev) => {
     if (!touched) setTouched(true)
-    const target = ev.target;
-    const val = target.type === 'checkbox' ? target.checked : target.value;
+    const val = ev.target.type === 'checkbox' ? ev.target.checked : ev.target.value;
     doValidation(val, touched)
     setIsChecked(val)
   }
 
+  const onKeyPress = (ev) => {
+    if (ev.key === 'Enter') {
```

Comment:
Note: While `KeyboardEvent.key` has support in every major browser (`n-1`), `IE11` implements an older version of the spec. [See this list of key identifiers supported in `IE9-11`](https://msdn.microsoft.com/en-us/ie/gg305568\(v=vs.94\)) and [this list for the modern spec](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values).

Reply:
We should also have a test to validate keyboard interaction does work as intended

Reply:
Thanks @bwalker8038 these are great points, and a nice list of things to work from! Timely too, I've just refactored `checked` to be `initialValue` so the field api is consistent with  itself. I'll get above changes in next! 🙏

Reply:
Am I missing something re IE?

![image](https://user-images.githubusercontent.com/142403/66763145-2f54cf80-ee5c-11e9-87fb-88bf888db158.png)
![image](https://user-images.githubusercontent.com/142403/66763164-38de3780-ee5c-11e9-8fec-b1b2967d75b9.png)

That tells me that `event.key` works on any IE we still support no? I was hoping to avoid coding in `13` and what not if possible. But, maybe I'm misreading so please advise if so—thanks!

Reply:
Ok, I think I'm understanding better...it's implemented but might be wrongly (caniuse had a better table then MDN as it showed more clearly it's supported but busted). For posterity, this helped me too:

https://webdev.ink/2018/02/10/Checking-for-Keyboard-Events-in-JavaScript-with-Cross-Browser-Support/

Damn, I thought I could finally just write a keypress listener without an `OR` 🤦‍♂

Reply:
@roblevintennis: I don't know why you're adding key listeners, and especially not in this PR, which is about something that seems pretty orthogonal.

>  but I'lll leave Enter too since it seems fine to have that as an option as well. I'll add some specs too and investigate an IE fallback too.

Are you making "Enter" a key listener for toggling the checkbox? Given how Enter is already used for submitting a form, that seems like a bad idea.

Reply:
I've removed the Enter but will leave the Space. I think it's fine to cleanup that piece of the a11y of the checkbox given I've just created it so recently and is not yet consumed anywhere.